src/main/frontend/src/components/FoodChainShow.js

```diff
@@ -21,8 +22,8 @@ const FoodChainShow = (props) => {
             .then(result => result.json())
             .then(foodChain => {
                 setFoodChain(foodChain)
-                setReviews(foodChain.reviewList.map(review => {
-                    return <div key={review.id}>{review.comment}<br></br>{review.rating}<br></br></div>
+                setReviews(foodChain.reviewList.map(review => { //McDonald's or w/e is unknown
+                    return <OneReview id={review.id} comment={review.comment} rating={review.rating}/>
```

Comment:
We should not be storing entire components in our state. it would be better to store the `reviewList` in state, and then do the `.map` to make `OneReview` components right before the rendering `return` above line 72